frontend/src/modules/results/connector.ts

```diff
@@ -211,6 +217,9 @@ export const getSearchResults = async (
             language,
             page_size: getGlobalConfig().searchResultsPageSize,
             page: pages.touristicEvents ?? undefined,
+            dates_before:
+              newDateFilter.dates_before === '' ? undefined : newDateFilter.dates_before,
```

Comment:
I don't know why you don't keep the string empty instead of defining it as undefined. Is there an edge case that i don't see?

Reply:
Oh yes, it's an omission. I replace it